dashboard/lib/service/dev_cocoon.dart

```diff
@@ -274,10 +274,11 @@ class DevelopmentCocoonService implements CocoonService {
     final int minAttempts = _minAttempts[status];
     final int maxAttempts = _maxAttempts[status];
     final int attempts = minAttempts + random.nextInt(maxAttempts - minAttempts + 1);
+    final int randomQueueTime = random.nextInt(1000 * 60 * 15);
```

Comment:
Generally, avoid random in fakes or tests. The reason we use it here is for the weights which we intentionally don't want to be deterministic.